StyleCop.Analyzers/StyleCop.Analyzers.Test/Helpers/DiagnosticResult.cs

```diff
@@ -65,7 +76,33 @@ public string Id
 
         public string Message
         {
-            get; set;
+            get
+            {
+                if (this.message != null)
+                    return this.message;
```

Comment:
This will trigger SA1503. There are several more instances below that do the same.